cfme/configure/configuration/diagnostics_settings.py

```diff
@@ -358,6 +373,30 @@ def in_server_collect_logs(self):
         )
 
 
+class ServerCollectLogsSlaveView(ServerCollectLogsView):
+    @property
+    def is_displayed(self):
+        slave_server = self.context['object'].appliance.server.slave_servers[0]
```

Comment:
**Question** What is the 0th element here?

Reply:
I'm adding inline comment there, it's the first slave server (out of one)